### ISSUE TITLE
Extract variant display names

### ISSUE THREAD

**Issue by pizzafroide** [papandreou/node-cldr#32]
Is there any reason why there is no `extractVariantDisplayNames` function? Something like:

```js
assert.equal(cldr.extractVariantDisplayNames('fr').VALENCIA, 'valencien');
```

**Comment by papandreou** [comment]
Hmm, no, no specific reason. Should be an easy thing to add. PR welcome :)

**Comment by papandreou** [comment]
Landed your patch in 4.1.0.
